packages/api/app/Controllers/Http/ResearcherController.js

```diff
@@ -0,0 +1,35 @@
+const User = use('App/Models/User');
+const Taxonomy = use('App/Models/Taxonomy');
+
+class ResearcherController {
+	async index({ request }) {
+		const filters = request.all();
+		const keywordTaxonomy = await Taxonomy.getTaxonomy('KEYWORDS');
+		const users = await User.query()
+			.where({ researcher: true })
+			.with('institution')
+			.with('areas')
+			.with('technologies', (builder) => {
+				builder.where('status', 'published');
+				builder.wherePivot('role', 'OWNER');
+				builder.with('terms', (builder2) => {
+					builder2.where('taxonomy_id', keywordTaxonomy.id);
+				});
+			})
+			.withResearcherFilters(filters)
+			.withParams(request, { skipRelationships: ['technologies'] });
+		const researchers = users.rows.map((user) => ({
+			full_name: user.toJSON().full_name,
+			institution: user.toJSON().institution.name,
+			areas: user.toJSON().areas,
```

Comment:
Acho que seria melhor dar um `toJSON()` antes do map, assim, acessaria todas as variáveis normalmente

```suggestion
		const researchers = users.toJSON().map((user) => ({
			full_name: user.full_name,
			institution: user.institution.name,
			areas: user.areas,
```